lib/mail/Personalization.php

```diff
@@ -240,6 +240,19 @@ public function setSendAt($send_at)
         $this->send_at = $send_at;
     }
 
+    /**
+     * Alias to SendAt()
+     * to have similar naming as other mail parts as used in Mail class
+     *
+     * @param SendAt $send_at SendAt object
+     *
+     * @throws TypeException
+     */
+    public function addSendAt($send_at)
+    {
+        $this->setSendAt($send_at);
```

Comment:
Does the `$send_at` variable should change into `$sendAt` because of the variable declaration consistency.

Reply:
It's named as in `setSendAt` method.

Reply:
I mean the variable declaration should use the `camel-case` style?

Reply:
Maybe. It's consistent with existing code in the class.

Reply:
Ok. I think it's okay to use `$send_at` variable because it's consistent for this `sendAt` method :).